File: client-development-app/src/Classes/GroceryType.ts
export default class GroceryType {

    type : string;
    color : string;

    constructor(type : string, color : string){
        this.type = type;
        this.color = color;
    }

    getType(){
        return this.type;
    }

    getColor(){
        return this.color;
    }
}

File: mobile-app/src/GroceryCart/app/src/main/java/com/example/grocerycart/GroceryType.java
package com.example.grocerycart;

import android.graphics.Color;

public class GroceryType {
    private String type;
    private String color;

    public GroceryType(String type,String color){
        this.type = type;
        this.color = color;
    }

    public String getType(){
        return type;
    }

    public String getColor(){
        return color;
    }

    public int getParsedColor(){
        return Color.parseColor(color);
    }

}
